scripts/distro/kubeadm/distro.sh

```diff
@@ -196,4 +196,28 @@ EOF
 function kubeadm_api_is_healthy() {
     curl --globoff --noproxy "*" --fail --silent --insecure "https://$(kubernetes_api_address)/healthz" >/dev/null
 }
+
+function kubeadm_conf_api_version() {
```

Comment:
Did you find that KUBERNETES_TARGET_VERSION_MINOR is ever unset or not correct to use for this function? What about the variable KUBERNETES_VERSION? In other code that is how we get the intended k8s version. I don't think we use kubeadm version.

Reply:
`KUBERNETS_TARGET_VERSION_*` vars always get parsed from the `KUBERNETES_VERSION` variable from [parse_kubernetes_target_version](https://github.com/replicatedhq/kURL/blob/main/scripts/common/utilbinaries.sh#L44). This function is invoked from the main scripts, `install.sh`, `join.sh`, etc.